### ISSUE TITLE
[reboot-proposal] Service setter

### ISSUE THREAD

**Issue by Yomguithereal** [jacomyal/domino.js#62]
The setter service option seems to be missing from the reboot. I don't remember if it was by design or not.

``` js
controller.registerService({
  url: '/users',
  setter: 'users'
});
```

Same thing as

``` js
controller.registerService({
  url: '/users',
  success: function(data) {
    this.update('users', data);
  }
});
```

The option is short to write and kinda handy. However, the `setter` name is not that good. Maybe we can find something better if we want to keep this option.

**Comment by jacomyal** [comment]
Yup, not sure if it was in the design either, but it still looks like a good idea.
